docs/usage.md

```diff
@@ -30,7 +30,7 @@ As shown in the examples below, the accepted format of the file is slightly diff
 
 #### With basecalling and demultiplexing
 
-##### Example `samplesheet.csv`
+##### Example `samplesheet.csv` for barcoded fast5 inputs
```

Comment:
any reason why you add this? Same for below

Reply:
I got the following error from the markdownlint check:
![Screenshot 2020-08-14 at 4 40 18 PM](https://user-images.githubusercontent.com/41866052/90230926-e7705180-de4c-11ea-9c9f-487f0b4fd766.png)

Reply:
there's this no duplicate heading requirement